.travis.yml

```diff
@@ -5,29 +5,43 @@ node_js:
   # so that your addon works for all apps
   - "4"
 
+sudo: false
```

Comment:
Does this not need to be `sudo: required`? I've had to do this a few times lately due to the recent Meltdown/Spectre changes in travis.

Reply:
fixed